extensions/wikia/CommunityPage/models/CommunityPageSpecialInsightsModel.class.php

```diff
@@ -4,18 +4,22 @@ class CommunityPageSpecialInsightsModel {
 	const INSIGHTS_MODULE_ITEMS = 3;
 	const INSIGHTS_CONFIG_SORT_TYPE_KEY = 'sortingType';
 	const INSIGHTS_CONFIG_PAGEVIEWS_KEY = 'displayPageviews';
+	const INSIGHTS_ICON = 'icon';
 	const INSIGHTS_MODULES = [
 		'popularpages' => [
 			self::INSIGHTS_CONFIG_SORT_TYPE_KEY => 'random',
```

Comment:
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png) Define a constant instead of duplicating this literal "random" 3 times. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://10.8.74.32:9000/coding_rules#rule_key=php%3AS1192)